packages/models-library/src/models_library/api_schemas_storage.py

```diff
@@ -274,6 +274,9 @@ class UploadedPart(BaseModel):
 class FileUploadCompletionBody(BaseModel):
     parts: list[UploadedPart]
 
+    def sort(self) -> None:
```

Comment:
I think you could have a validator that automatically sorts such as

```python

@validator("parts")
@classmethod
def ensure_sorted(cls, value: list[UploadedPart]) -> list[UploadedPart]:
    return value.sort()
```

Reply:
i agree

Reply:
Yes I realize I can use the validators to do this. I didn't want to do this because I didn't want to mix data validation with data modification. I felt by putting it in a separate method it was more clear that the data from the user actually gets modified.
But if you really prefer(?) I can add it to the validation step.

Reply:
It is a common practice pydantic validators are used for both parsing and pre/post procressing inputs. 

The idea of pydantic is to create a dataset with some guarantees. If you are always going to use this list sorted, then you should add that post-processing in the validator function. 

Note that even the annotations you are using have some default parsing (i.e. pre-processing) capabilities (e.g. strings get parsed as int if int is expected). 

In this casen it would be as if you have annotated it as `sorted_list[UploadedPart]`  (note that `sorted_list` does not exist, it is just illustrative)

Reply:
OK, makes sense. I will adapt